hotel_daily_service/wizard/hotel_daily_service_wizard.py

```diff
@@ -0,0 +1,76 @@
+import logging
+from datetime import timedelta
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from dateutil import parser
+except (ImportError, IOError) as err:
+    _logger.debug(err)
+
+
+class HotelDailyServiceLineWizard(models.TransientModel):
+    _name = "hotel.daily.service.line.wizard"
+    _description = "Hotel Daily Service Line"
+
+    @api.model
+    def _default_hotel_service_id(self):
+        breakfast = self.env.ref("hotel_daily_service.hotel_service_breakfast")
+        return breakfast
+
+    hotel_folio_id = fields.Many2one(
+        comodel_name="hotel.folio", string="Folio"
```

Comment:
No need to add `comodel_name`, simply you can use fields.Many2one('hotel.folio',

Reply:
I know, but we prefer to use explicit keywords :)